### PR TITLE
Enable the GCS reporter (again)

### PR DESCRIPTION
**Pull request by Katharine**
Turn on the GCS reporter.

I'll monitor this and handle anything that goes wrong, but not until after my next meeting, so holding until then

/hold
/cc @michelle192837 @fejta

### CODE REVIEW

**Comment by michelle192837** [comment]
Fingers crossed!

File: prow/cluster/crier_deployment.yaml [hunk 1]
```diff
@@ -41,6 +41,8 @@ spec:
         - --github-endpoint=https://api.github.com
         - --slack-workers=1
         - --slack-token-file=/etc/slack/token
+        - --gcs-worker=1
+        - --gcs-credentials-file=/etc/service-account/service-account.json
```
Comment (fejta) [inline prow/cluster/crier_deployment.yaml]: boo :)

**Comment by Katharine** [comment]
I made a typo. Fixed now.

/hold cancel

**Pull request merged**